cookiecutter.json

```diff
@@ -4,16 +4,16 @@
     "github_org": "my-org",
     "__source_path": "src/{{cookiecutter.__project_slug}}/schema/{{cookiecutter.__project_slug}}.yaml",
     "project_description": "This is the project description.",
-    "full_name": "Harshad Hegde",
-    "email": "hhegde@lbl.gov",
+      "full_name": "My Name",
+      "email": "my-name@my-org.org",
     "__author": "{{cookiecutter.full_name}} <{{cookiecutter.email}}>",
     "license": ["MIT", "BSD-3", "GNU GPL v3.0", "Apache Software License 2.0"],
     "google_analytics_gtag": "G-2SYBSJVZ23",
     "main_schema_class": "Person",
     "create_python_classes": ["Yes", "No"],
     "use_schemasheets": [
-        "Yes",
-        "No (next two questions are ignored)"
+        "No (next two questions are ignored)",
```

Comment:
I have already fixed this and the PR was merged to main this morning but the `(next two questions are ignored)` should not be present. This is because the whole string `"No (next two questions are ignored)"` becomes a value and if we need to add an if condition e.g. `{% if cookiecutter.use_schemasheets == "No" %}` won't work. It'll have to be `{% if cookiecutter.use_schemasheets == "No (next two questions are ignored)" %}` which is weird. You might want to pull down from main to this branch.